miden-lib/asm/sat/prologue.masm

```diff
@@ -369,17 +299,17 @@ end
 #!  - The account data is invalid.
 #!  - Any of the consumed notes do note exist in the note db.
 #!
-#! Stack:       [BH, acct_id, IAH, NC]
+#! Stack:        [BH, acct_id, IAH, NC]
 #! Advice stack: [acct_id, ZERO, ZERO, nonce, AVR, ASR, ACR,
-#!               num_cn,
-#!               CN1_SN, CN1_SR, CN1_IR, CN1_VR,
-#!               cn1_na,
-#!               CN1_A1, CN1_A2, ...
-#!               CN2_SN,CN2_SR, CN2_IR, CN2_VR,
-#!               cn2_na,
-#!               CN2_A1, CN2_A2, ...
-#!               ...]
-#! Output:      []
+#!                num_cn,
+#!                CN1_SN, CN1_SR, CN1_IR, CN1_VR,
+#!                cn1_na,
+#!                CN1_A1, CN1_A2, ...
+#!                CN2_SN,CN2_SR, CN2_IR, CN2_VR,
+#!                cn2_na,
+#!                CN2_A1, CN2_A2, ...
+#!                ...]
```

Comment:
Now that we have support for `advice_map` - let's create an issue to refactor this to get values from the `advice_map` rather than `advice_stack`.

Reply:
I've made an issue for this [here](https://github.com/0xPolygonMiden/miden-base/issues/72) but I'm not sure if it makes sense to do this.  It seems to me that moving the data to be in the `advice_map` and then having the assembly code instruct the prover to place the data on the `advice_stack` just adds additional steps and increases the complexity of the miden code?  I think it's preferable to leave it as it is and work under the assumption that the prover has already arranged data on the `advice_stack` correctly.

Reply:
Ah - makes sense! I was thinking of the `advice_map` because for libraries, we don't want to rely on the `advcie_stack` as controlling it becomes difficult. But this is an executable program - so, using `advice_stack` is actually simpler. I'll close that issue.